### ISSUE TITLE
merge.rspec() tweak for row-vs-col merges

### ISSUE THREAD

**Issue by thomased** [rmaia/pavo#175]
Wondering if `merge.rspec()` could be tweaked to allow for row- or column-wise merges (probably via a new argument, since `by = ` specifies the column to be used for merging). I'm thinking of UV-VIS + NIR merges, where you've got separate measures of spectral regions separates but want to row-merge them into one full range spec. Would need to think about how to handle spectral gaps etc., but shouldn't be too tough.

**Comment by Bisaloo** [comment]
I think it would be better to use another function since it would differ a lot from `merge.data.frame`, which is the function most users will be familiar with.

I was thinking of using `rbind.rspec` but it's not ideal either since it may discard rows in case of wavelength range overlap. Maybe the most reasonable option is a completely new function such as `stitch.rspec`.

**Comment by thomased** [comment]
Yeah fair enough. `merge` and `stitch` — I could see something like that working.

**Comment by Bisaloo** [comment]
`combine` could be another option.

**Comment by Azem1hyseni** [comment]
I'm so interesting to  be part of your  group  in rock filer foundation  , i have one small  office in  Kosovo but we have large impact  in development and more .
